src/CommonITILObject.php

```diff
@@ -9215,7 +9215,8 @@ public static function getDataToDisplayOnKanban($ID, $criteria = [])
             $data = [
                 'id'        => $data['id'],
                 'name'      => $data['name'],
-                'category'  => $category_names[$data['itilcategories_id']] ?? '',
+                'category'  => $data['itilcategories_id'],
+                'category_name' => $category_names[$data['itilcategories_id']] ?? '',
```

Comment:
There is a duplicated logic here. On line 9205, the translated category name is fetched (one request for one category) and then, on lines 9349->9411, the translated categories names are fetched again (one request for all categories) to replace categories ids by their name.

Some cleaning should be done. I guess #16084 would not have been necessary if the lines 9349->9411 were working as expected.